CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/MyHabits.java

```diff
@@ -0,0 +1,85 @@
+package com.example.cmput_301_project;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class MyHabits extends AppCompatActivity implements HabitFragments.OnFragmentInteractionListener{
+    private RecyclerView recyclerView;
+    List<Habit> modelList;
+    private Button removeButton;
+    private Button cancelButton;
+    private RecyclerViewAdapter adapter2;
```

Comment:
I can't seem to find another adapter item, so why is this the second? Perhaps a more descriptive name of what this adapter is.